src/javascript/app_2/Stores/Modules/Trading/Helpers/__tests__/durations.js

```diff
@@ -0,0 +1,135 @@
+import { expect }    from 'chai';
+import * as Duration from '../duration.js';
+import BinarySocket  from '_common/base/socket_base';
+
+describe('buildDurationConfig', () => {
+    const contract = {
+        barrier_category: "euro_atm",
+        barriers: 0,
+        contract_category: "callput",
+        contract_category_display: "Up/Down",
+        contract_display: "Higher",
+        contract_type: "CALL",
+        exchange_name: "FOREX",
+        expiry_type: "daily",
+        market: "forex",
+        max_contract_duration: "365d",
+        min_contract_duration: "1d",
+        sentiment: "up",
+        start_type: "spot",
+        submarket: "major_pairs",
+        underlying_symbol: "frxAUDJPY"
+    }
+
+    const durations = {
+        min_max: {
+            spot: {
+                daily: {
+                    min: 86400,
+                    max: 31536000
+                },
+            }
+        },
+        units_display: {
+            spot: [{
+                text: "days",
+                value: "d"
+            }]
+        }
+    }
+
+    it('Returns correct value when durations is not passed', () => {
+        expect(Duration.buildDurationConfig(contract)).to.eql(durations);
+    });
+
+    it('Returns correct value when durations passed', () => {
+        expect(Duration.buildDurationConfig(contract, durations)).to.eql(durations);
+    });
+
+});
+
+describe('convertDurationUnit', () => {
+    it('Returns null if the arguments are empty value', () => {
+        expect(Duration.convertDurationUnit('','','')).to.be.null;
+    });
+
+    it('Returns null if there is no arguments', () => {
+        expect(Duration.convertDurationUnit()).to.be.null;
+    });
+
+    it('Returns correct value convert day to second', () => {
+        expect(Duration.convertDurationUnit("365", "d", "s")).to.eql(31536000);
+    });
+
+    it('Returns correct value convert minute to second', () => {
+        expect(Duration.convertDurationUnit("5", "m", "s")).to.eql(300);
+    });
+
+    it('Returns correct value convert day to minute', () => {
+        expect(Duration.convertDurationUnit("1", "d", "m")).to.eql(1440);
+    });
+
+    it('Returns correct value convert second to minute', () => {
+        expect(Duration.convertDurationUnit("180", "s", "m")).to.eql(3);
+    });
+
+    it('Returns correct value convert minute to day', () => {
+        expect(Duration.convertDurationUnit("2880", "m", "d")).to.eql(2);
+    });
+
+    it('Returns correct value convert second to day', () => {
+        expect(Duration.convertDurationUnit("86400", "s", "d")).to.eql(1);
+    });
```

Comment:
I think we should add a couple of test cases where the argument is there but it is not a valid one. e.g. what should happen when we pass `convertDurationUnit('alpha', 'm', 'd')`? Do we expect null or an error?